src/ParseServer.js

```diff
@@ -361,7 +359,20 @@ function injectDefaults(options: ParseServerOptions) {
       ])
     );
 
-    options.protectedFields = { _User: { '*': userSensitiveFields } };
+    if (!('protectedFields' in options)) {
+      options.protectedFields = [];
```

Comment:
Not sure it matters but protected fields is an object so would  `= {}` be more accurate?

Reply:
yeah.

Reply:
this isn't covered fyi,